src/engine/bufferscalers/enginebufferscalerubberband.h

```diff
@@ -15,7 +16,12 @@ class EngineBufferScaleRubberBand : public EngineBufferScale {
   public:
     explicit EngineBufferScaleRubberBand(
             ReadAheadManager* pReadAheadManager);
-    ~EngineBufferScaleRubberBand() override;
+
```

Comment:
while you're at it, you can make this class `final` to aid the compiler in "devirtualizing" some function calls.

Reply:
Good idea, added.